lms/djangoapps/teams/views.py

```diff
@@ -0,0 +1,37 @@
+"""
+View methods for the course team feature.
+"""
+
+from django.shortcuts import render_to_response
+from opaque_keys.edx.keys import CourseKey
+from courseware.courses import get_course_with_access, has_access
+from django.http import Http404
+from django.conf import settings
+from student.models import CourseEnrollment
+
+
+def teams_dashboard(request, course_id):
+    """
+    Renders the teams dashboard, which is shown on the "Teams" tab.
+
+    Raises a 404 if the user is not registered for the course, or if the teams feature is not enabled.
```

Comment:
What does it do if the course_id is invalid or refers to a course that doesn't exist?